modules/skeletonRetriever/app/conf/config.ini

```diff
@@ -10,11 +10,11 @@ tracking-threshold          50
 time-to-live                1.0
 
 [depth]
-enable                      true
-kernel-size                 24
-iterations                  12
-min-distance                1.0
-max-distance                4.0
+enable                      false
+;kernel-size                 24
+;iterations                  12
+;min-distance                1.0
+;max-distance                4.0
```

Comment:
Is this a trick to better highlight that this section is disabled?
Asking because `enable false` is sufficient.

Reply:
It was a trick to keep not of the depth filtering parameters, which could have been useful, but maybe it's better to remove them.

Reply:
You can just leave them there. They won't be simply used because of `enable false`.